dunstify: add page (#3501)
tldr-pages/tldr: pages/linux/dunstify.md

# dunstify

> A notification tool that is an extension of notify-send, but has more features based around dunst.

> > Works with all options that work for notify-send.

Reviewer: Could we add a more information link below this line?

Author: Added the command to get more information at the bottom!

Reviewer 2: I only found a github page [here](https://github.com/dunst-project/dunst), not sure there is a online document outside.

Author: Yes, the parent project is dunst, the notification manager, but there is very little documentation about dunstify, which is the notification sending command

Reviewer: No problem - thanks for checking @Jfeatherstone :smiley_cat:

> - Show a notification with title "Title" and message "Message":

Reviewer: Perhaps this will read better?
```suggestion
- Show a notification with a given title and message:
```

`dunstify {{"Title"}} {{"Message"}}`

> - Show a notification with specified urgency (options: "low", "normal", "high"):

Reviewer: Perhaps this will read better?
```suggestion
- Show a notification with specified urgency:
```
Note the next suggestion too, which goes with this one.

> `dunstify {{"Title"}} {{"Message}}" -u high`

Reviewer: ```suggestion
`dunstify {{"Title"}} {{"Message}}" -u {{low|normal|high}}`
```

> - Show a message with a message ID of 123 (will overwrite previous messages with the same ID):

Reviewer: Perhaps this will read better?
```suggestion
- Specify a message ID (overwrites any previous messages with the same ID):
```

> `dunstify {{"Title"}} {{"Message"}} -r 123`

Reviewer: `123` here should be wrapped in the token syntax `{{` `}}`.
```suggestion
`dunstify {{"Title"}} {{"Message"}} -r {{123}}`
```

# dunstify

> > A notification tool that is an extension of notify-send, but has more features based around dunst.

Reviewer 3: Let's put notify-send within backticks

Reviewer: Oops, looks like I missed this!

> > Works with all options that work for notify-send.

Reviewer 3: Here also
